ArcGISRuntimeSDKQt_QMLSamples/Features/FeatureLayer_DefinitionExpression/FeatureLayer_DefinitionExpression.qml

```diff
@@ -77,22 +77,62 @@ Rectangle {
         }
         spacing: 5
 
+        Label {
+
+            text: "Current feature count: " + updateFeatureCount();
+        }
```

Comment:
❌ 

I don't think this will work. My guess is that this will execute one time only. A function has no bindable attributes, so no ability to "notify" this expression there is an updated value.

But fortunately in the QML Api, the feature table's `queryFeatureCountResult` property does notify, so it will automatically update this label whenever there is an updated value.

```suggestion
        Label {
            text: "Current feature count: " + featureTable.queryFeatureCountResult
        }
```

Reply:
This does not seem to be work, the feature count always says "0" but the way I have it implemented seems to work as expected

Reply:
Are you calling `featureTable.queryFeatureCount(...)` anywhere? That's how `queryFeatureCountResult` will get updated.